openapi-spec/swagger.yaml

```diff
@@ -867,95 +979,135 @@ components:
               description: >-
                 Prefix of object name, only those objects which name have this
                 prefix will be choosed.
+              example: photos/
+            Tag:
+              type: array
+              description: Tag of object name.
+              items:
+                $ref: '#/components/schemas/KV'
     Job:
       type: object
       properties:
         Id:
-          description: 'The plan ID, generate by server'
+          description: The plan ID, generated by server
           type: string
-          example: 5bd6f273b0bdf90001dc91c3
+          example: '5bd6f273b0bdf90001dc91c3'
         TenantId:
           description: The tenant UUID in a multi-tenancy environment.
           type: string
-          example: 1ff92505-3efb-45bd-8e2d-764a1ce0fa6b
+          example: '94b280022d0c4401bcf3b0ea85870519'
+        UserId:
+          description: The User UUID in a multi-tenancy environment.
+          type: string
+          example: '558057c4256545bd8a307c37464003c9'
+        TriggerType:
+          description: The migration triggering operation type.
+          type: string
+          example: manual
+          enum:
+            - auto
+            - manual
         Type:
           description: >-
             The plan type. Till now only migration support. New types will be
             added in the future.
           type: string
           example: migration
-          enum:
-            - migration
+          enum: *JOBS
         PlanName:
           description: The name of plan job belongs to.
           type: string
+          example: plan001
         PlanId:
           description: The id of plan job belongs to.
           type: string
+          example: '5eaa81fd6f384d000116d1d4'
         SourceLocation:
           description: The source data migrate or replicate from.
           type: string
+          example: bucket001
         DestLocation:
           description: The target data migrate or replicate to.
           type: string
+          example: bucket002
         CreateTime:
           description: The time job is created.
           type: integer
+          example: 1588233982
         StartTime:
           description: The time job started to run.
           type: integer
+          example: 158823398
         EndTime:
           description: The time job ended.
           type: integer
+          example: 1588233983
         Status:
           description: The status of job.
           type: string
           enum:
             - pending
-            - rnning
+            - running
             - succeed
```

Comment:
```suggestion
            - succeess
```

Reply:
Again This is with the current Model Design. We don't want to confuse Users for different Status

Reply:
Ok. I just see the code that the status is returned succeed. Let's keep it as it is now then...We will fix code and then update here too later

Reply:
No Change Done.